freeipa.spec.in

```diff
@@ -78,8 +78,8 @@
 %global ds_version 1.4.3.16-12
 %global selinux_policy_version 3.14.3-107
 %else
-# DNA interval enabled
-%global ds_version 2.0.5-1
+# version supporting LMDB and lib389.cli_ctl.dblib.run_dbscan utility
+%global ds_version 2.1.0
```

Comment:
The current DS version in main, the version supporting LMDB by default, is 3.0.

Reply:
The new code needs to be compatible with Berkeley DB and LMDB and just needs the api `lib389.cli_ctl.dblib.run_dbscan` which was introduced in 2.1.0. IPA doesn't want to require (yet) LMDB support.